libs/SalesforceSDKCore/SalesforceSDKCore/CSFSalesforceActionTests.m

```diff
@@ -0,0 +1,83 @@
+//
+//  CSFSalesforceActionTests.m
+//  SalesforceSDKCore
+//
+//  Created by Jonathan Arbogast on 2/16/17.
+//  Copyright © 2017 salesforce.com. All rights reserved.
+//
+
+@import XCTest;
```

Comment:
@jarbogast Move this file [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/tree/dev/libs/SalesforceSDKCore/SalesforceSDKCoreTests), where all the other tests live.